.travis.yml

```diff
@@ -45,8 +45,8 @@ install:
 
 # Create test databases
 before_script:
-  - npm install --global -gulp-cli
-  - npm install
+  - npm i -g gulp
+  - npm i
```

Comment:
@Manorlds-Eaglespark remove this. A PR resolving this was raised.

Reply:
@AGMETEOR, that was resolved. Kindly check https://github.com/andela/wger-kronos/pull/26/files